doc/creating_gsdml_files.rst

```diff
@@ -292,13 +294,19 @@ has these attributes:
 * ``SupportedProtocols="SNMP;LLDP"`` Conformance class B must support SNMP.
 * ``PTP_BoundarySupported="true"``
 * ``DCP_BoundarySupported="true"``
-* ``DCP_HelloSupported="true"`` Often used for fast startup.
+* ``DCP_HelloSupported="true"`` Used for fast startup. Then the
+  ``PowerOnToCommReady`` attribute must be set as well.
```

Comment:
Remove "Then the". Instead on single line:
``PowerOnToCommReady`` attribute must be set  if DCP_HelloSupported is set to true